src/openrct2/ride/ride_ratings.c

```diff
@@ -621,21 +622,25 @@ static void ride_ratings_calculate_value(rct_ride *ride)
 
 	sint32 monthsOld = gDateMonthsElapsed - ride->build_date;
```

Comment:
I would move this right below `if (!gCheatsDisableRideValueAging) {` since it's only used when the cheat is not active.